libcudacxx/test/libcudacxx/cuda/mdspan/host_accessor.runfail.cpp

```diff
@@ -0,0 +1,27 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the libcu++ Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES.
+//
+//===----------------------------------------------------------------------===//
+// UNSUPPORTED: nvrtc
+
+#include <cuda/mdspan>
+
+#include "test_macros.h"
+
+int main(int, char**)
+{
+#if !defined(__CUDA_ARCH__)
```

Comment:
Please move into a test function and conditionally call that with `NV_IF_TARGET`